Makefile

```diff
@@ -37,14 +37,14 @@ vortex_acoustics : $(OBJ) | $(DRIVER_ARCHIVE)
 	$(CXX) $(CFLAGS) $^ -o $@ $(LDFLAGS)
 
 $(BUILD_DIR) :
-	mkdir -p $@/driver
+	mkdir -p
```

Comment:
What does mkdir -p do when the directory name is left blank?

Reply:
Have no clue! Bug I didn't notice. Should revert this, however I am terrible with makefiles and cannot get one workinh

Reply:
Fair haha. Can still merge without it, before testing on the actual board starts :wink:

Reply:
The makefile is not that important as the code can be compiled elsewhere. It is nice to get working for testing the code on the computer etc.